src/api/app/components/build_results_monitor_component.html.haml

```diff
@@ -0,0 +1,34 @@
+.accordion#build-results-monitor
+  - package_names.each do |package_name|
+    .accordion-item
+      .accordion-header
+        .accordion-button.p-3.collapsed{ "aria-controls" => "collapse-#{package_name.gsub(':', '_')}",
+                                         "aria-expanded" => "false", "data-bs-target" => "#collapse-#{package_name.gsub(':', '_')}",
+                                         "data-bs-toggle" => "collapse",
+                                         type: "button" }
+          .w-100.d-flex.justify-content-between.pe-3
+            = package_name
+            %span.text-end
+              - Buildresult.avail_status_values.each do |status|
+                - result_count_per_status = results_per_package_and_status(package_name, status).count
+                - if result_count_per_status > 0
+                  = render BuildStatusBadgeComponent.new(status: status, text: "#{result_count_per_status}")
+    .accordion-collapse.border.p-4.pt-1.collapse{ id: "collapse-#{package_name.gsub(':', '_')}" }
+      .list-group.list-group-flush
+        - repository_names.each do |repository_name|
+          .list-group-item.d-flex.justify-content-between
+            = link_to(helpers.word_break(repository_name, 22),
+                      project_package_repository_binaries_path(project_name: project_name,
+                                                               package_name: package_name,
+                                                               repository_name: repository_name),
+                      title: "Binaries for #{repository_name}")
+            %span.text-end
+              - results_per_package_and_repository(package_name, repository_name).each do |result|
+                - url = live_build_log_url(result[:status], project_name, package_name, repository_name, result[:architecture])
+                = render BuildStatusBadgeComponent.new(status: result[:status],
```

Comment:
I see the point of _organizing more_ the view, but we need to consider corner cases as well as normal cases. There are corner cases where I don't really see arches into columns to work well always, because possible arches are up to a dozen or even 20.